common/source/docs/common-modalai-voxl.rst

```diff
@@ -43,6 +43,7 @@ The minimum steps to get the camera working include
 
 - `Setup adb <https://docs.modalai.com/setup-adb/>`__ on an Ubuntu workstation
 - Connect the camera to the Ubuntu workstation, start and adb shell and `run voxl-configure-vision-px4 <https://docs.modalai.com/voxl-vision-px4-installation/>`__ (Note: you may simply press <enter> when asked for an IP address)
+- run "systemctl start voxl-mavlink-server" to configure the mavlink server to start automatically
```

Comment:
should be enable if you want it to always start on boot. Start will just start it once

Reply:
Yes , agree on that

Reply:
@khancyr @patrickpoirier51,

Great, thanks very much for the correction.  I will change this to, "systemctl enable voxl-mavlink-server"